modules/jdbc/src/main/java/org/testcontainers/jdbc/ContainerDatabaseDriver.java

```diff
@@ -313,4 +318,16 @@ public static void killContainer(String jdbcUrl) {
             }
         }
     }
+
+    /**
+     * Utility method to get an instance of a database container given its JDBC URL.
+     * @param jdbcUrl the JDBC URL of the container instance to get
+     * @return an instance of database container or <code>null</code> if no container associated with JDBC URL
+     */
+    @VisibleForTesting
+    public static JdbcDatabaseContainer getContainer(String jdbcUrl) {
```

Comment:
There are lot's of places in the existing codebase, where we're violating this rule. Since it's not a really useful rule if you want to use with package-scope, I'd say we simply deactivate this pattern in Codacy. @rnorth are you okay with this?